### PR TITLE
Angular Colorpicker

### PR DESCRIPTION
**Pull request by clado**
An angular colorpicker for use with angular views. Fashioned after [jQuery Minicolors] (https://github.com/claviska/jquery-miniColors/).

### CODE REVIEW

File: angular-minicolors/angular-minicolors.js [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+        var swatch = $compile(inputTemplate)(scope)
+        element.after(swatch)
+
+        var defaults = {
+          lowercase: true,
+          defaultColor: '#ffffff',
+          expand: true
+        }
+
+        var options = _.defaults(_.pick(scope, ['lowercase', 'defaultColor', 'expand']), defaults)
```
Comment (rib3) [inline angular-minicolors/angular-minicolors.js]: Array should be fine then.

**Comment by rib3** [comment]
CR: :+1: let's move forward with this.

**Comment by clado** [comment]
QA can be found [here](https://github.com/sportngin/sport_admin/pull/1556)

**Comment by clado** [comment]
[QA](https://github.com/sportngin/sport_admin/pull/1556)

**Pull request merged**